### PR TITLE
Make qdrant the default vector db

### PR DESCRIPTION
**Pull request by imartinez**
- Docs updated
- Move qdrant out of optional dependencies
- Move chrome to optional dependencies
- Default settings.yaml to load a QdrantLocal

### CODE REVIEW

**Comment by imartinez** [comment]
Could you take a quick look @Anush008 ?

**Review by Anush008 — APPROVED** [review]
Looks good! 👍

**Comment by Anush008** [comment]
The tests seem to be instantiating multiple `QdrantLocal` instances concurrently, causing the failure.
We could use a Qdrant cloud instance for the tests.

File: fern/docs/pages/manual/vectordb.mdx [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
+PrivateGPT supports [Qdrant](https://qdrant.tech/) and [Chroma](https://www.trychroma.com/) as vectorstore providers. Qdrant being the default.
 
-### Qdrant configuration
-
-To enable Qdrant, set the `vectorstore.database` property in the `settings.yaml` file to `qdrant` and install the `qdrant` extra.
+In order to select one or the other, set the `vectorstore.database` property in the `settings.yaml` file to `qdrant` or `chroma`.
 
-```bash
-poetry install --extras qdrant
+```
```
Comment (lopagela) [inline fern/docs/pages/manual/vectordb.mdx:6]:
```suggestion
```yaml
```

File: tests/server/ingest/test_ingest_service.py
Comment (lopagela) [inline tests/server/ingest/test_ingest_service.py:1]: Not to be addressed in this PR:

Some work is required on the tests, to be able to tests more integrations, like here, keeping our tests with Chroma (while still addings tests on `qdrant`)

**Review by lopagela — DISMISSED** [review]
LGTM

Minor: putting the syntax name next to triple back quotes in `.md(x)` is enabling colorization of the output

**Review by pabloogc — APPROVED** [review]

File: private_gpt/server/ingest/ingest_service.py [hunk 2]
```diff
@@ -207,7 +207,12 @@ def delete(self, doc_id: str) -> None:
         )
 
         # Load the index with store_nodes_override=True to be able to delete them
-        index = load_index_from_storage(self.storage_context, store_nodes_override=True)
+        index = load_index_from_storage(
+            storage_context=self.storage_context,
+            service_context=self.ingest_service_context,
+            store_nodes_override=True,  # Force store nodes in index and document stores
+            show_progress=True,
+        )
```
Comment (lopagela) [inline private_gpt/server/ingest/ingest_service.py:215]: FYI, I will be changing this in my PR that speeds up document ingestion

**Review by lopagela — APPROVED** [review]

**Review by Anush008 — APPROVED** [review]

**Pull request merged**